render/src/test/java/org/codice/imaging/nitf/render/JpegTest.java

```diff
@@ -96,4 +96,33 @@ public void checkJpegMaskRender2() throws IOException, URISyntaxException {
         renderer.render(mockImageSegment, targetGraphic);
     }
 
+    @Test
+    public void checkJpeg2000MaskRender() throws IOException, URISyntaxException {
```

Comment:
Can you make a new test file (like Jpeg2000Test or similar) and move this test into that file? In NITF, JPEG and JPEG2000 are quite distinct, and would prefer to keep the tests split.

Ideally there would be a proper render test, but recognise that will be hard without rel. test data.

Can you duplicate this test to do 2x3 (as well as the current 3 x 2) - row/column confusion has been a source of bugs in the past.